demo-openid/src/Issuer.ts

```diff
@@ -110,6 +126,25 @@ function getCredentialRequestToCredentialMapper({
       }
     }
 
+    if (credentialConfigurationId === universityDegreeCredentialMdoc.id) {
+      const holderKey = await agentContext.dependencyManager
+        .resolve(MdocService)
+        .getKeyFromMdocCredentialHolderBinding(agentContext, holderBinding)
+
+      return {
+        credentialSupportedId: universityDegreeCredentialMdoc.id,
+        format: ClaimFormat.MsoMdoc,
+        docType: universityDegreeCredentialMdoc.doctype,
+        issuerCertificate: trustedCertificates[0],
+        holderKey,
```

Comment:
Can't we return the holder binding like we do with sd-jwt?

Reply:
We can, but then we need to decide internally how to e.g. extract a key from a did-holder biding with mdoc, which I think should be up to the user?

Reply:
Hmm how to extract the key is generally straigtforward. You resolve the did, find the key with the key id, transform it to a key instance (like you do now). I think we should just not allow did holder binding when using mdoc?

Reply:
hmm but I understand now what you mean, it will break the API. Okay let's keep it like this 👍 

Could we also add the `key` to the binding / credential request mapper? Then you at least don't have to call the mdoc service.